### DISCUSSION TITLE
Delay tooltip deactivation after trigger loses hover

### DISCUSSION THREAD

**Discussion by nstolpe** [radix-ui/primitives#2633] [Help]
I'm working on a project that makes heavy use of raxix-ui primitives and has tooltip feature with a requirement to persist for x amount of time after the trigger loses hover. I've looked through the docs and examples and haven't been able to find anything, but I'm hoping someone knows of a way to implement this (or can point me to some docs I missed).

**Comment by benoitgrelard** [comment]
I don't believe this is possible, as it would cause other UX issues when keeping tooltips open after they've been blurred.

**Reply by nstolpe** [reply]
Ok, thanks for getting back to me.
